### DISCUSSION TITLE
Is it possible to use torchaudio wav2vec generated alignment for `lhotse cut trim-to-alignments`?

### DISCUSSION THREAD

**Discussion by JinZr** [lhotse-speech/lhotse#1161] [Q&A] [answered]
As titled.

**Comment by desh2608** [comment, marked as answer]
I think it should be possible, since that workflow also stores alignments within the supervisions. Are you getting any errors?

**Comment by JinZr** [comment]
Dear Desh,

Ah not a bug I encountered, I would like to try it but I'm not sure if this
will work out.

Best

**Reply by pzelasko** [reply]
As Desh mentioned, it should "just work."
